src/content/docs/dev/amplifier/chain-integration/relay-messages/gmp-api.mdx

```diff
@@ -0,0 +1,445 @@
+# GMP API
```

Comment:
can we simply link to the notion doc where this content came from? that notion doc was created and maintained by axelarons. i thought about porting them over here but figured it's better to leave in notion and just continue to link to it.

Reply:
Ya will do this^. Wasn't sure how we felt sharing notion stuff externally but will do that if we're okay with it